lib/eslint-rails/warning.rb

```diff
@@ -1,6 +1,6 @@
 module ESLintRails
   class Warning
-    attr_reader :filename, :rule_id, :message, :line, :column, :node_type
+    attr_reader :filename, :message, :line, :column, :node_type
```

Comment:
We should also remove the `message` `attr_reader` here since we're overwriting the reader below.